sys-fs/lvm2/lvm2-2.02.173.ebuild

```diff
@@ -66,7 +66,9 @@ PATCHES=(
 	"${FILESDIR}"/${PN}-2.02.99-locale-muck.patch #330373
 	"${FILESDIR}"/${PN}-2.02.70-asneeded.patch # -Wl,--as-needed
 	"${FILESDIR}"/${PN}-2.02.139-dynamic-static-ldflags.patch #332905
+	"${FILESDIR}"/${PN}-2.02.171-static-libm.patch #617756
 	"${FILESDIR}"/${PN}-2.02.172-static-pkgconfig-libs.patch #370217, #439414 + blkid
+	"${FILESDIR}"/${PN}-2.02.173-static-libm.patch #617756
```

Comment:
I don't see 2.0.2.173-static-libm.patch anywhere in this commit?

Reply:
Sorry, one copy & paste to much :/ Fixed not. There is only the .171 patch.